client/src/constants/routes.js

```diff
@@ -0,0 +1,8 @@
+export const LOGIN_PAGE = '/login';
+export const HOME_PAGE = '/';
+export const COHORT_PAGE = '/cohorts';
+export const ADD_COHORT = '/add-cohort';
+export const STUDENTS_PAGE = '/students';
+export const ADD_STUDENT = '/add-student';
+export const PROJECTS_PAGE = '/projects';
+export const ADD_PROJECT = '/add-project';
```

Comment:
this route need modify, you need to put /admin before each route Except login route
like this
```js
export const LOGIN_PAGE = '/login';
export const HOME_PAGE = '/admin';
export const COHORT_PAGE = '/admin/cohorts';
export const ADD_COHORT = '/admin/add-cohort';
export const STUDENTS_PAGE = '/admin/students';
export const ADD_STUDENT = '/admin/add-student';
export const PROJECTS_PAGE = '/admin/projects';
export const ADD_PROJECT = '/admin/add-project';
```